proto/testbed.proto

```diff
@@ -100,6 +100,7 @@ message Port {
     string pmd_regex = 5;
   }
 
+  // Name of group to which this port belongs; ports may be grouped for link aggregation
```

Comment:
From the perspective of a user who reads this doc, I would want to know what to expect: (1) what does it do, and (2) are there any caveats. For (1), I would say "configure the port to be a member of this Link Aggregation Group", and for (2) I think it's fair to say that this is specific to a KNE container operation, which means this is not observed on a hardware testbed. The wording "ports may be grouped" sounds non-committal to me.

Reply:
Let's workshop this. Instead of "ports may be grouped for link aggregation" how is "ensures the ports are eligible for link aggregation by the test"? Clearer?

Reply:
@anjan-keysight @liulk Thoughts on whether my suggestions were clearer, or do you have alternative suggestions?

Reply:
I would make a minor change, "In KNE, ensures ... by the test" so the fact it's a no-op elsewhere is upfront.

Reply:
A new version of testbed.proto has been merged with updated documentation of the group field.